assets/stylesheets/_components.scss

```diff
@@ -331,7 +331,6 @@
 @import 'my-sites/invites/invite-accept-logged-in/style';
 @import 'my-sites/invites/invite-accept/style';
 @import 'my-sites/media-library/style';
```

Comment:
If we move out this style for `media-library/style` then clear your cache and load any page in Calypso prior to a section that includes the media routes, you get this "Choose File" input near the Add button:

<img width="606" alt="screen shot 2017-10-17 at 7 50 30 am" src="https://user-images.githubusercontent.com/7240478/31668632-f6d2a31e-b30f-11e7-9a95-18c9e7987a90.png">

As a result, I left the style here in `_components`.

Reply:
Good catch. The problem lies with the `MediaLibraryUploadButton` component, which is embedded [in the sidebar](https://github.com/Automattic/wp-calypso/blob/57619e165a42087d6eb111a8c90cd380f4cbf5d5/client/my-sites/sidebar/manage-menu.jsx#L201-L207). We could import `my-sites/media-library/style` from the new `media` CSS bundle, and just import the [stylesheet](https://github.com/Automattic/wp-calypso/blob/2380716c75c6bb05273b49e2d77a22f72f25fc56/client/my-sites/media-library/upload-button.scss) for this button here in `_components.scss`. This would move most of the CSS code to the `media` section, and fix this particular display issue.